.github/workflows/main-build-and-deploy.yml

```diff
@@ -120,3 +120,45 @@ jobs:
       - name: Swap to the production slot
         run: |
           az webapp deployment slot swap --resource-group ${{ vars.RESOURCE_NAME_PREFIX }}-rg --name ${{ vars.WEBAPP_NAME }} --slot ${{ vars.WEBAPP_SLOT_NAME }} --target-slot production
+
+  deploy-to-production:
+    runs-on: ubuntu-22.04
+    needs: [create-and-publish-image, deploy-to-dev]
```

Comment:
The array needs deploy-to-dev switching out for the staging job

Reply:
Just for additional context, if this was left as is, the deployment to staging and production would happen at the same time. Changing this to deploy-to-staging will cause it to wait :-)

Reply:
Well spotted, fixing now.